paper/MocoPaperAppendix.tex

```diff
@@ -186,11 +186,11 @@
          & V_{L,k} \leq V_k(t_0, t_f, y_0, y_f, x_{0}, x_{f}, \lambda_0, \lambda_f, p, S_{b,k}) \leq V_{U,k} && \textrm{boundary constraints} \\
          & \quad\quad S_{b,k} = \int_{t_0}^{t_f} s_{b,k}(t, y, x, \lambda, p)\,dt \quad k = 1, \ldots, K\\
          & g_{L} \leq g(t, y, x, \lambda, p) \leq g_{U} && \textrm{path constraints} \\
-         & y_{0,L} \leq y_0 \leq y_{0,U} \quad\quad y_{f,L} \leq y_f \leq y_{f,U} && \textrm{initial and final states} \\
-         & x_{0,L} \leq x_0 \leq x_{0,U} \quad\quad x_{f,L} \leq x_f \leq x_{f,U} && \textrm{initial and final controls} \\
          \mbox{with respect to} \quad
          & t_0 \in [t_{0,L}, t_{0,U}] && \textrm{initial time} \\
          & t_f \in [t_{f,L}, t_{f,U}] && \textrm{final time} \\
+         & y_0 \in [y_{0,L}, y_{0,U}] \quad\quad y_f \in [y_{f,L}, y_{f,U}] && \textrm{initial and final states} \\
+         & x_0 \in [x_{0,L}, x_{0,U}] \quad\quad x_f \in [x_{f,L}, x_{f,U}] && \textrm{initial and final controls} \\  
```

Comment:
In the continuous optimal control problems, I like putting these as constraints because `y_0` and `x_0`, etc. are not separate variables; they are included within `y(t)` and `x(t)`.